certify-core/src/main/java/io/mosip/certify/core/spi/TokenService.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.core.spi;
+
+import io.mosip.certify.core.dto.OIDCTransaction;
+import io.mosip.certify.core.exception.CertifyException;
+import io.mosip.certify.core.exception.NotAuthenticatedException;
+import org.json.simple.JSONObject;
+
+public interface TokenService {
```

Comment:
Do we need TokenService interface in certify service? If only some constants are used, let's take them to the Constants file.

Reply:
moved them to constants file